experts-client/lib/experts-client.js

```diff
@@ -30,41 +31,42 @@ export { localDB };
 * @classdesc Aggie Experts Client API provide methods to import and access Aggie Experts data.
 */
 export class ExpertsClient {
-  
+
   /**
    * @constructor
    * Accepts a cli object with options from a commander program.
    */
   constructor(cli = {}) {
 
-    console.log('ExpertsClient constructor');
-    console.log(cli);
+    // console.log('ExpertsClient constructor');
+    // console.log(cli);
 
-    // Accept CLI options for these values if they are provided. Defaults are set in the .env file.
-    cli.source ??= [process.env.EXPERTS_FUSEKI_ENDPOINT + process.env.EXPERTS_FUSEKI_PROFILE_SOURCE];
-    cli.bind ??= process.env.EXPERTS_FUSEKI_PROFILE_BIND;
-    cli['construct@'] ??= process.env.EXPERTS_FUSEKI_PROFILE_CONSTRUCT;
-    cli.iamAuth ??= process.env.EXPERTS_IAM_AUTH;
-    cli.iamEndpoint ??= process.env.EXPERTS_IAM_ENDPOINT;
-    cli.fusekiAuth ??= process.env.EXPERTS_FUSEKI_AUTH;
+    // Should process.env be part of the library, or the CMD line? and maybe
```

Comment:
So you can override the defaults for as many config values as you want.

Reply:
But I guess the env to cli fill can happen in the CLI script?

Reply:
But if the values are typically consistent, it would be nice to be able to just import the experts-client module and use methods without doing that part.